packages/web/src/pages/rewards-page/components/modals/ChallengeRewardsModal/AudioMatchingRewardsModalContent.tsx

```diff
@@ -170,37 +157,36 @@ export const AudioMatchingRewardsModalContent = ({
   }
 
   return (
-    <Flex column gap='2xl'>
+    <Flex column gap='2xl' justifyContent='space-between' h='100%'>
       {isMobile ? (
-        <>
+        <Flex column gap='2xl'>
           {progressDescription}
-          <Paper column shadow='flat' w='100%' css={{ overflow: 'hidden' }}>
-            <Flex justifyContent='center'>{progressReward}</Flex>
-            {progressStatusLabel}
-          </Paper>
+          {progressReward}
+          {progressStatusLabel}
           {renderCooldownSummaryTable()}
-        </>
+        </Flex>
       ) : (
-        <>
-          <Paper shadow='flat' direction='column'>
-            <Flex justifyContent='center'>
-              {progressDescription}
-              {progressReward}
-            </Flex>
-            {progressStatusLabel}
-          </Paper>
+        <Flex column gap='2xl'>
+          <Flex alignItems='center' justifyContent='space-between' gap='2xl'>
+            {progressDescription}
+            {progressReward}
+          </Flex>
+          {progressStatusLabel}
           {renderCooldownSummaryTable()}
-        </>
-      )}
-      {challenge?.claimableAmount && challenge.claimableAmount > 0 ? null : (
-        <Button
-          variant='secondary'
-          fullWidth
-          {...ctaButtonProps[challengeName]}
-          onClick={handleClickCTA}
-        />
+        </Flex>
       )}
-      {errorContent}
+      {/* Hack alert, move this button down closer to the submit button */}
+      <Box css={{ marginBottom: -16 }}>
+        {challenge?.claimableAmount && challenge.claimableAmount > 0 ? null : (
```

Comment:
think this can just be `!!challenge?.claimableAmount`. 
- the value can't be less than zero
- if the value is 0, then the condition will evalutate to false
- if the value is greater than 0, the condition will evaluate to true

Reply:
yeah down i think ts complains about undefined > 0, but can look through it again!